CHANGELOG.md

```diff
@@ -212,6 +212,14 @@
    * ADDED: "has_transit_tiles" & "osm_changeset" to verbose status response [#4062](https://github.com/valhalla/valhalla/pull/4062)
    * ADDED: time awareness to CostMatrix for e.g. traffic support [#4071](https://github.com/valhalla/valhalla/pull/4071)
    * UPDATED: transifex translations [#4102](https://github.com/valhalla/valhalla/pull/4102)
+   * ADDED: Added `exclude_tolls` request parameter [#4524](https://github.com/valhalla/valhalla/pull/4524)
+   * ADDED: Added `exclude_bridges` request parameter [#4524](https://github.com/valhalla/valhalla/pull/4524)
+   * ADDED: Added `exclude_tunnels` request parameter [#4524](https://github.com/valhalla/valhalla/pull/4524)
+   * ADDED: Added `service_limits.allow_hard_exclusions` server config parameter [#4524](https://github.com/valhalla/valhalla/pull/4524)
+   * ADDED: Added `exclude_highways` request parameter [#4524](https://github.com/valhalla/valhalla/pull/4524)
+   * ADDED: Added `exclude_ferries` request parameter [#4524](https://github.com/valhalla/valhalla/pull/4524)
+   * ADDED: Added `exclude_ferry` location search filter [#4524](https://github.com/valhalla/valhalla/pull/4524)
+   * ADDED: Added `exclude_toll` location search filter [#4524](https://github.com/valhalla/valhalla/pull/4524)
```

Comment:
nit: can we make this a single changelog entry just listing all the new things added. its ok to have multiple sentences describing whats going on

Reply:
No problem, its done